srcpkgs/pypy3.5/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'pypy3.5'
+pkgname=pypy3.5
+version=7.0.0
+revision=1
+wrksrc="pypy3.5-v${version}-src"
+hostmakedepends="pkg-config python python-cffi"
+makedepends="bzip2-devel gdbm-devel libffi-devel liblzma-devel
+ libressl-devel ncurses-devel sqlite-devel tk-devel zlib-devel"
+short_desc="JIT-enabled implementation of Python 3.5 written in RPython"
+license="MIT"
+homepage="http://pypy.org/"
+distfiles="https://bitbucket.org/pypy/pypy/downloads/pypy3.5-v${version}-src.tar.bz2"
+checksum=b2ddb0f45cb4e0384fb498ef7fcca2ac96c730b9000affcf8d730169397f017f
+nocross="Tries to execute cross-compiled code"
+
+do_build() {
+	cd pypy/goal
+	python ../../rpython/bin/rpython --opt=jit --cc=${CC} \
+		--make-jobs=$XBPS_MAKEJOBS
+	PYTHONPATH=../.. ./pypy3-c ../tool/build_cffi_imports.py
+}
+
+pre_install() {
```

Comment:
Why have pre_install if you are defining do_install() ?

the pre and post functions are when you don't want to mess with build_style defined do_ functions

Reply:
Fair enough, that was my misunderstanding.